barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -178,15 +177,6 @@ template <typename Flavor> class SumcheckProver {
         , round(multivariate_n)
         , partially_evaluated_polynomials(multivariate_n){};
 
-    /**
```

Comment:
This proxy function is the only reason why the instances classes had to be in the sumcheck module. They properly belong with the decider prover and verifier.